tests/unit/unit_tests_err.cpp

```diff
@@ -0,0 +1,61 @@
+#include <limits.h>
+#include "gtest/gtest.h"
+
+extern "C" {
+#include "src/redismodule.h"
+#include "src/err.h"
+}
+
+namespace {
+
+void Alloc_Reset() {
+    RedisModule_Alloc = malloc;
+    RedisModule_Realloc = realloc;
+    RedisModule_Calloc = calloc;
+    RedisModule_Free = free;
+    RedisModule_Strdup = strdup;
+}
```

Comment:
Add a new file, such as RedisGraph `rmalloc.h` that has this function in it.
Next, for each unit test create a c++ class that calls `Alloc_Reset`
```
#include "gtest.h"

#ifdef __cplusplus
extern "C" {
#endif
#include "path/to/rmalloc.h"
#ifdef __cplusplus
}
#endif

class MyTestClass:
	public ::testing::Test {
  protected:
	static void SetUpTestCase() { // Use the malloc family for allocations
		Alloc_Reset();
	}
};
```

Reply:
thank you @DvirDukhan . proceed with the changes.